helm/hpcc/values.schema.json

```diff
@@ -709,6 +734,42 @@
         },
         "resources": {
           "$ref": "#/definitions/resources"
+        },
+        "annotations": {
+          "type": "object",
+          "additionalProperties": { "type": "string" }
+        },
+        "labels": {
+          "type": "object",
+          "additionalProperties": { "type": "string" }
+        },
+        "service": {
+          "description": "Service properties",
+          "type": "object",
+          "required": [ "servicePort", "visibility" ],
+          "properties": {
+            "port": {
+              "type": "integer",
+              "description": "The local port used by the pod",
+              "default": 8880
+            },
+            "servicePort": {
+              "type": "integer",
+              "description": "The port that this service will be exposed on"
+            },
+            "visibility": {
+              "type": "string",
+              "description": "Should this service be exposed outside the cluster, locally or to the internet"
```

Comment:
"enum": [ "cluster", "local", "global" ],?

Reply:
It was originally, but it's now possible for people to add their own visibility classes if they want

Reply:
Should the restriction be removed for the other place where visibility is defined?

Reply:
Yes - well spotted. I will push another fix